src/vs/workbench/services/editor/browser/editorOverrideService.ts

```diff
@@ -452,7 +455,8 @@ export class EditorOverrideService extends Disposable implements IEditorOverride
 		}
 		// Map the editors to quickpick entries
 		registeredEditors.forEach(editor => {
-			const isActive = currentEditor ? editor.editorInfo.describes(currentEditor) : false;
+			const currentViewType = (currentEditor as IContributedEditorInput).viewType ?? DEFAULT_EDITOR_ASSOCIATION.id;
```

Comment:
@bpasero Here's an example of that common casting pattern where I have an `IEditorInput` but need the viewType from it (even if it's undefined). So I cast it to an editor input that has a viewtype.

Reply:
I will reply in my review comment on this.